vite-project/src/scouter/ScouterQuery.tsx

```diff
@@ -9,54 +9,39 @@ interface ScouterQueryProps {
   list?: string[];
 }
 
+export const localStorageTabName = "Queries/";
+
 const ScouterQuery: React.FC<ScouterQueryProps> = ({
   name,
   queryType,
   required,
   list,
 }) => {
   function renderInput() {
-    const queryName = `Queries/${name}`;
-    const initialValue = localStorage.getItem(queryName) || "";
     switch (queryType) {
       case "counter":
-        return <CounterQuery name={queryName} initialValue={initialValue} />;
+        return <CounterQuery name={name} />;
       case "checkbox":
-        return (
-          <CheckboxQuery
-            name={queryName}
-            required={required || false}
-            initialValue={initialValue}
-          />
-        );
+        return <CheckboxQuery name={name} required={required || false} />;
       case "list":
         return (
-          <ListQuery
-            name={queryName}
-            required={required}
-            initialValue={initialValue}
-            list={list ? list : []}
-          />
+          <ListQuery name={name} required={required} list={list ? list : []} />
         );
       case "radio":
         return (
-          <RadioQuery
-            name={queryName}
-            required={required}
-            initialValue={initialValue}
-            list={list ? list : []}
-          />
+          <RadioQuery name={name} required={required} list={list ? list : []} />
```

Comment:
^^^
and here it is also required
think about weather "|| false" is required, if its not remove it, if it is, add it where needed